client/components/mma/cancel/cancellationSaves/ContinueMembershipConfirmation.tsx

```diff
@@ -1,28 +1,34 @@
 import { css } from '@emotion/react';
 import { space } from '@guardian/source-foundations';
-import {
-	Button,
-	Stack,
-	SvgArrowRightStraight,
-} from '@guardian/source-react-components';
+import { Button, Stack } from '@guardian/source-react-components';
 import { useContext } from 'react';
 import { useNavigate } from 'react-router';
-import { Heading } from '../../shared/Heading';
+import { cancellationFormatDate } from '../../../../../shared/dates';
+import type {
+	PaidSubscriptionPlan} from '../../../../../shared/productResponse';
+import {
+	getMainPlan
+} from '../../../../../shared/productResponse';
 import { ProgressIndicator } from '../../shared/ProgressIndicator';
 import type {
-	CancellationPageTitleInterface} from '../CancellationContainer';
+	CancellationContextInterface} from '../CancellationContainer';
 import {
-	CancellationPageTitleContext
+	CancellationContext
 } from '../CancellationContainer';
-import { buttonLayoutCss } from './SaveStyles';
+import { headingCss, stackedButtonLeftLayoutCss } from './SaveStyles';
 
 export const ContinueMembershipConfirmation = () => {
 	const navigate = useNavigate();
+	const cancellationContext = useContext(
+		CancellationContext,
+	) as CancellationContextInterface;
+	const membership = cancellationContext.productDetail;
 
-	const pageTitleContext = useContext(
-		CancellationPageTitleContext,
-	) as CancellationPageTitleInterface;
-	pageTitleContext.setPageTitle('Membership confirmation');
+	const mainPlan = getMainPlan(
+		membership.subscription,
+	) as PaidSubscriptionPlan;
+
+	const chargeThroughDate = `${mainPlan.chargedThrough}`;
```

Comment:
don't think this needs to be put into an interpolated string const, can just be referenced directly below